### PR TITLE
refact: add to cart

### PR DESCRIPTION
**Pull request by kaioken**
- hotfix add to cart

### CODE REVIEW

File: test/cart.test.ts [hunk 1]
```diff
@@ -5,6 +5,19 @@ beforeAll(async () => {
 });
 
 describe('Test the Commerce', () => {
+/*     it('add to cart', async () => {
```
Comment (jerlyrosa) [inline test/cart.test.ts:8]: why is this commented?

**Review by jerlyrosa — APPROVED** [review]

File: src/modules/commerce/cart/index.ts [hunk 2]
```diff
@@ -5,12 +5,12 @@ import { CartData, CartItemData, CartItemInput } from '../../../types/commerce';
 export class Cart {
   constructor(protected client: ClientType) { }
 
-  public async addToCart(input: CartItemInput): Promise<CartItemData> {
+  public async addToCart(input: CartItemInput[]): Promise<CartItemData> {
```
Comment (buttercubz) [inline src/modules/commerce/cart/index.ts:8]: must be a mayor like: 0.16.0 because this is a breaking change

File: test/cart.test.ts (hunk 1 above)
Comment (kaioken) [reply test/cart.test.ts]: @jerlyrosa done

File: src/modules/commerce/cart/index.ts (hunk 2 above)
Comment (kaioken) [reply src/modules/commerce/cart/index.ts:8]: @buttercubz done

**Review by jerlyrosa — APPROVED** [review]

**Review by AudryLopez — APPROVED** [review]

**Pull request merged**